products/batch_exports/backend/temporal/destinations/utils.py

```diff
@@ -0,0 +1,21 @@
+import datetime as dt
+
+
+def _get_query_timeout(data_interval_start: dt.datetime | None, data_interval_end: dt.datetime) -> float:
```

Comment:
do you think it would be better to remove the leading underscore here? Not sure if the same for you, but the convention I'm used to is that underscore methods are only used within the same module, whereas this one feels like it's 'public' as it's imported by other modules

Reply:
I don't think it matters much anyways. Python does not have public/private methods. But I'll remove it.